lib/std/meta.zig

```diff
@@ -1334,3 +1334,13 @@ test "shuffleVectorIndex" {
     testing.expect(shuffleVectorIndex(6, vector_len) == -3);
     testing.expect(shuffleVectorIndex(7, vector_len) == -4);
 }
+
+/// Returns whether `error_union` contains an error.
+pub fn isError(error_union: anytype) bool {
+    return if(error_union) |_| false else |_| true;
```

Comment:
Doesn't zig fmt add a space after the 'if'? I thought the stdlib was run through zig fmt?

Reply:
That is blocking on #8219.